src/core/lib/iomgr/event_engine_shims/endpoint.cc

```diff
@@ -171,6 +189,9 @@ class EventEngineEndpointWrapper {
   std::unique_ptr<grpc_event_engine_endpoint> eeep_;
   std::atomic<int64_t> refs_{1};
   std::atomic<int64_t> shutdown_ref_{1};
+#ifdef GRPC_POSIX_SOCKET_TCP
+  absl::AnyInvocable<void(absl::StatusOr<int>)> on_release_fd_ = nullptr;
```

Comment:
`= nullptr` is unnecessary with AnyInvocable.

Reply:
Removed.